cli/src/commands/invite/claim.rs

```diff
@@ -89,6 +147,19 @@ async fn step0(
     Ok(ctx)
 }
 
+/// Step 0.5: choose recipient
+fn shamir_pick_recipient(
+    ctx: &ShamirRecoveryClaimPickRecipientCtx,
+) -> anyhow::Result<ShamirRecoveryClaimInitialCtx> {
+    let recipients = ctx.yet_to_contact_recipients();
+    let human_recipients: Vec<_> = recipients.iter().map(|r| r.human_handle.clone()).collect();
```

Comment:
If `ShamirRecoveryRecipient` implemented `fmt::Display` we would not need to create a new vec :thinking:

Reply:
I understand